sharktank/sharktank/models/llm/llm.py

```diff
@@ -257,7 +257,9 @@ def __init__(
         super().__init__(theta)
 
         attention_kernel = (
-            "decomposed" if config.hp.model_arch == "grok" else config.attention_kernel
+            "decomposed"
+            if config.hp.model_arch in ["grok", "gpt-oss"]
+            else config.attention_kernel
```

Comment:
Following up on https://github.com/nod-ai/shark-ai/pull/2377#discussion_r2411437012

Please have this get set in llm_configs.py.